README.md

```diff
@@ -450,6 +452,39 @@ You need to provide the IP address and port of at least one member in your clust
     hazelcast::client::ClientConfig config;
     config.getNetworkConfig().addAddress(hazelcast::client::Address("your server ip", 5701 /* your server port*/));
 ```
+### 1.4.3. Client System Properties
+
+While configuring your C++ client, you can use various system properties provided by Hazelcast to tune its clients. These properties can be set programmatically through `config.SetProperty` or by using an environment variable.
```

Comment:
In go it is called `config` this should be `clientConfig` as it is called `clientConfig` in C++.

Reply:
This is the variable name (you can set anything) but not the class name, hence it should not matter.